app/config/config.yml

```diff
@@ -121,3 +121,6 @@ swp_updater:
     version_class: %swp_updater.version.class%
     client:
         base_uri: http://updates.sourcefabric.org
+
+swp_bridge:
+    base_uri: 'http://example.com:5050'
```

Comment:
let's change it for ```http://superdesk.dev:5050``` - we will make it configurable later from ui bot for now would be better to just create host like that locally and make it working without modyfing content. Example.com isn't best name in this case.